atuin-client/src/settings.rs

```diff
@@ -1,374 +1,110 @@
+mod behaviour;
+pub mod display;
+mod input;
+mod stats;
+mod sync;
+mod time;
+
+use ::time as time_lib;
+
 use std::{
     collections::HashMap,
-    convert::TryFrom,
-    fmt,
     io::prelude::*,
     path::{Path, PathBuf},
     str::FromStr,
 };
 
 use atuin_common::record::HostId;
-use clap::ValueEnum;
 use config::{
     builder::DefaultState, Config, ConfigBuilder, Environment, File as ConfigFile, FileFormat,
 };
-use eyre::{bail, eyre, Context, Error, Result};
+use eyre::{eyre, Context, Result};
 use fs_err::{create_dir_all, File};
 use parse_duration::parse;
 use regex::RegexSet;
 use semver::Version;
 use serde::Deserialize;
-use serde_with::DeserializeFromStr;
-use time::{
-    format_description::{well_known::Rfc3339, FormatItem},
-    macros::format_description,
-    OffsetDateTime, UtcOffset,
-};
+use time_lib::{format_description::well_known::Rfc3339, Duration, OffsetDateTime};
 use uuid::Uuid;
 
```

Comment:
Exporting all of the moved types saves making changes elsewhere for now...

Reply:
pretty reasonable imo!